support-frontend/app/controllers/RedemptionController.scala

```diff
@@ -80,6 +82,24 @@ class RedemptionController(
       })
   }
 
+  def displayError(redemptionCode: RedemptionCode, error: String)(implicit request: AuthRequest[Any]): Result = {
+    SafeLogger.error(scrub"An error occurred while trying to process redemption code - ${redemptionCode}. Error was - ${error}")
```

Comment:
we should also do a cloudwatch putMetric at this point so we know it's broken.  We have one already for warnings (non critical failures on server side) so it might be worth piggy backing on that

Reply:
Discussed offline, we're going to revisit tracking and alarms in a separate PR